src/pages/tokens/[id].js

```diff
@@ -125,45 +135,66 @@ function TokenPage() {
     pollInterval: 1800000,
   });
 
-  const chartDatas = tokenDayDatas.reduce(
```

Comment:
why are we changing this also?

Reply:
https://github.com/voltfinance/voltage-analytics/pull/1#discussion_r841658743